k8s/operators/src/pool/main.rs

```diff
@@ -257,11 +257,17 @@ async fn main() -> anyhow::Result<()> {
         env!("CARGO_PKG_VERSION"),
     );
 
-    let fmt_style = matches.get_one::<FmtStyle>("fmt-style").unwrap();
+    let fmt_style = matches.get_one::<String>("fmt-style").unwrap().as_str();
+    let fmt_style = match fmt_style {
+        "json" => FmtStyle::Json,
+        "compact" => FmtStyle::Compact,
+        _ => FmtStyle::Pretty,
+    };
```

Comment:
leave as it was before, and update arg with `.value_parser(clap::value_parser!(FmtStyle))`